OutOfSchool/OutOfSchool.WebApi/Services/AreaAdminService.cs

```diff
@@ -166,7 +166,20 @@ public async Task<SearchResult<AreaAdminDto>> GetByFilter(AreaAdminFilter filter
         {
             var regionAdminDto = await regionAdminService.GetByUserId(currentUserService.UserId).ConfigureAwait(false);
             filter.InstitutionId = regionAdminDto.InstitutionId;
-            var childrenIds = await codeficatorService.GetAllChildrenIdsByParentIdAsync(filter.CATOTTGId);
+            var childrenIds = await codeficatorService.GetAllChildrenIdsByParentIdAsync(regionAdminDto.CATOTTGId);
+            if (filter.CATOTTGId != 0)
+            {
+                if (childrenIds.Contains(filter.CATOTTGId))
```

Comment:
This method includes parent Id to the result list, so it works right in this case

Reply:
> This method includes parent Id to the result list, so it works right in this case

I looked again on code, and thought that it would be better to add equality check, so `GetAllChildrenIdsByParentIdAsync` not called in case you mentioned

Reply:
Perfect! As this was my second question )